ArchiSteamFarm/Steam/Integration/ArchiHandler.cs

```diff
@@ -277,6 +277,44 @@ public sealed class ArchiHandler : ClientMsgHandler {
 		return response.Result == EResult.OK;
 	}
 
+	[PublicAPI]
+	public async Task<(ulong ChatGroupID, ulong DefaultChatID)?> GetClanChatInfo(ulong steamID) {
```

Comment:
Make this one named `GetClanChatRoomInfo` and return `CClanChatRooms_GetClanChatRoomInfo_Response` instead, there's no reason to limit the response just to two properties out of all, especially since you're making this public API.

Reply:
Thought it would be bad to expose Protobuf classes to PublicAPI

Anyway, fixed. Thx for review

Reply:
This function calls low-level internal Steam API which produces given response - if it breaks in any way, that's on Steam and not us. ArchiHandler should not have any very advanced logic, it's only a very simple middleman between Steam and ASF/plugins so we don't need to ask ourselves which endpoint to call with what request and response.